src/web-auth/index.js

```diff
@@ -182,8 +182,11 @@ WebAuth.prototype.validateAuthenticationResponse = function(options, parsedHash,
   var state = parsedHash.state;
   var transaction = this.transactionManager.getStoredTransaction(state);
   var transactionState = options.state || (transaction && transaction.state) || null;
+
   var transactionStateMatchesState = transactionState === state;
```

Comment:
Won't this be `true` for both values of `null`? e.g. `null === null`  -> `transactionStateMatchesState=true`?